### PR TITLE
Pull request from feature branch in develop

### PR DESCRIPTION
**Pull request by aristides1000**
Through the following pull request, we request that the possibility of joining the feature branch in the development branch be studied, knowing the knowledge acquired in the authentication of users on different platforms through the use of Rails

### CODE REVIEW

**Review by MkrtichSargsyan — CHANGES_REQUESTED** [review]
## _Status:_ Changes Requested :recycle:

## Hello guys


## To Highlight :tada:


1. You have descriptive git commit messages.
2. Your readme file looks professional
3. Tt displays Flash messages for unsuccessful actions


However, there are some changes that you need to make in order to get your project approved

- [x] Kindly make the development branch the default branch

- [x] pull request description is not descriptive enough

- [x] Images are not showing

err

### routes

- [x] It creates the application routes using resources :posts, only: [:new, :create, :index] [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png)

### PostsController

- [x] it makes sure that the user_id for the post is the one from the signed_in user [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_create_requirement.png)

    ex. @post = current_user.posts.build(post_params)

 Its important to not pass user_id in params (security) 
<hr>

Please, make the REQUIRED changes and submit your request again! :grinning:
Good luck. 

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  devise_for :users, :controllers => { registrations: 'registrations' }
+  resources :posts
```
Comment (MkrtichSargsyan) [inline config/routes.rb]: It creates the application routes using resources :posts, only: [:new, :create, :index]

**Review by george-shammar — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUIRED** :recycle: :recycle: :construction_worker_man: 

Well done, team for work done so far on this project.

**Highlights**
:heavy_check_mark: You've implemented some of the required changes from the previous review and your repo also looks professional.
Good job :+1: 

- [x] However, it appears that while you were implementing the changes, you broke something else.

Immediately I signed up, I encountered this error: :point_down: :point_down: 
Screenshot 2021-07-14 at 15-35-31 ActionView SyntaxErrorInTemplate at
As a result, I couldn't verify that you've implemented the feature specific change from the last review.

Please make the required change and submit another CRR when you're ready.

**Cheers** :smile: **With respect** :tophat:

**Comment by ad9311** [comment]
Fix issue redering /posts/new.html.erb

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

### Happy Coding ☺️

## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

**Pull request merged**